FlowCrypt/Controllers/Compose/ComposeViewController.swift

```diff
@@ -328,7 +328,7 @@ extension ComposeViewController {
         to: [String],
         cc: [String] = [],
         bcc: [String] = [],
-        atts: [SendableMsg.Att] = []
+        atts: [SendableMsg.Attribute] = []
```

Comment:
These are `Att` as for attachment. If it's confusing then call them `Attachment`.

Reply:
Will fix this, thank you